__mocks__/v8-profiler-lambda.js

```diff
@@ -1,8 +1,18 @@
+const { Readable } = require('stream');
+
+class MockReadable extends Readable {
+  constructor(options) {
+    super(options);
+  }
+  _read(size) { this.push(null); }
```

Comment:
comment that implementations of Readable push null to end the stream per nodejs docs